server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/BallotManifestDownloadByCounty.java

```diff
@@ -88,16 +83,13 @@ public String endpoint(final Request the_request, final Response the_response) {
           Main.GSON.toJson(matches, bw);
           bw.flush();
         } catch (final IOException e) {
-          status = HttpStatus.INTERNAL_SERVER_ERROR_500;
-          result = "Unable to stream response";
+          serverError(the_response, "Unable to stream response");
         }
       }
     } else {
-      status = HttpStatus.NOT_FOUND_404;
-      result = "Invalid county ID specified";
+      dataNotFound(the_response, "Invalid county ID specified");
```

Comment:
This text could be improved. In other cases, "invalid" means "malformed" or "can't be parsed as the right type", and we call `invariantViolation()`. Here, the ID is well-formed, but doesn't have a representation in the database, so we (rightly) call `dataNotFound()`. Maybe we should change the other uses of "invalid" to "malformed"?